microovn/ovn/cmd/commands.go

```diff
@@ -167,7 +183,14 @@ func VSCtl(ctx context.Context, s state.State, args ...string) (string, error) {
 		return "", err
 	}
 
-	return shared.RunCommandContext(ctx, "ovs-vsctl", args...)
+	// try command 3 times if it is failing
+	for attempts := 0; attempts < 3; attempts++ {
+		output, err := shared.RunCommandContext(ctx, "ovs-vsctl", append(args, "--timeout", "30")...)
```

Comment:
I suspect that the test failures are because the `--timeout 30` is added at the end of the command. This may be messing up the expectations of the `ovs-vsctl`.
I don't think there's a `prepend` functions for slices in GO, so you might need to do something like this https://github.com/canonical/microovn/blob/96c619d8e3173ecb338bca1bd059490bdf6fa96b/microovn/ovn/cmd/commands.go#L176-L183